CHANGELOG.md

```diff
@@ -6,8 +6,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+:warn: **Breaking Change ahead!** :warn:
```

Comment:
`:warn:` is not Markdown or any format whatsoever and just looks strange. I think the bold `Breaking Change ahead` should be sufficient.

Reply:
I also would set the `Breaking Change` line between the `## [Unreleased]` and the `### Changed` line to make it more prominent.